### PR TITLE
CB-20769 Propagate additional autoscaling trigger failure details to …

### PR DESCRIPTION
**Pull request by aakashshankar**
…the activity console

See detailed description in the commit message.

### CODE REVIEW

File: autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/FreeIpaCommunicator.java [hunk 1]
```diff
@@ -32,18 +31,8 @@ public FreeIpaCommunicator(UserV1Endpoint userV1Endpoint) {
     @Retryable(value = Exception.class, maxAttempts = 5, backoff = @Backoff(delay = 10000))
     public SyncOperationStatus synchronizeAllUsers(SynchronizeAllUsersRequest request) {
         LOGGER.info("Invoking freeIpa user sync request: {}", request);
-        String envCrn = request.getEnvironments().iterator().next();
-        SyncOperationStatus lastSyncStatus = ThreadBasedUserCrnProvider.doAsInternalActor(
+        return ThreadBasedUserCrnProvider.doAsInternalActor(
```
Comment (sidseth) [inline autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/FreeIpaCommunicator.java:34]: Why this change ? (in the current patch)

File: autoscale/src/main/resources/schema/app/20230201124421_CB-20769_Modify_the_data_type_of_statusreason_in_history_table.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+-- // CB-20769 Modify the data type of statusreason in history table
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE history ALTER COLUMN status_reason TYPE TEXT;
```
Comment (sidseth) [inline autoscale/src/main/resources/schema/app/20230201124421_CB-20769_Modify_the_data_type_of_statusreason_in_history_table.sql:4]: Unrelated change ?

File: autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/FreeIpaCommunicator.java (hunk 1 above)
Comment (aakashshankar) [reply autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/FreeIpaCommunicator.java:34]: Added this change because the IPA service throws an error when user sync is invoked for the first time in an env

File: autoscale/src/main/resources/schema/app/20230201124421_CB-20769_Modify_the_data_type_of_statusreason_in_history_table.sql (hunk 2 above)
Comment (aakashshankar) [reply autoscale/src/main/resources/schema/app/20230201124421_CB-20769_Modify_the_data_type_of_statusreason_in_history_table.sql:4]: Character varying type is insufficient when attempting to store longer messages in the history entity resulting in PSQL exceptions. Changed it to `text` type here

**Review by Vatsal23082000 — APPROVED** [review]
